src/pages-and-resources/teams/TeamSetEditor.jsx

```diff
@@ -0,0 +1,208 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import { Button, Form, TransitionReplace } from '@edx/paragon';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import CollapsableEditor from '../../generic/CollapsableEditor';
+import messages from './messages';
+
+/**
+ * Team sizes enum
+ * @enum
+ * @type {{MIN: number, MAX: number, DEFAULT: number}}
+ */
+export const TeamSizes = {
+  DEFAULT: 50,
+  MIN: 1,
+  MAX: 500,
+};
+
+/**
+ * Team types enum
+ * @enum
+ * @type {{PRIVATE_MANAGED: string, PUBLIC_MANAGED: string, OPEN: string}}
+ */
+export const TeamSetTypes = {
+  OPEN: 'open',
+  PUBLIC_MANAGED: 'public_managed',
+  PRIVATE_MANAGED: 'private_managed',
```

Comment:
Move into constant file

Reply:
There should be a constants folder that contains the whole MFE constants. It would be centralized and easy to debug.